src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -407,21 +402,19 @@ public String getQuestionResultStatisticsCsv(
         }
 
         for (Map.Entry<String, String> entry : sortedMap.entrySet()) {
-            contribFragments += entry.getValue();
+            contribFragments.append(entry.getValue());
         }
-        
-        String csv = "";
-        
-        //Header
-        csv += "In the points given below, an equal share is equal to 100 points. ";
-        csv += "e.g. 80 means \"Equal share - 20%\" and 110 means \"Equal share + 10%\"." + Const.EOL;
-        csv += "Claimed Contribution (CC) = the contribution claimed by the student." + Const.EOL;
-        csv += "Perceived Contribution (PC) = the average value of student's contribution as perceived by the team members." + Const.EOL;
-        csv += "Team, Name, Email, CC, PC, Ratings Recieved" + Const.EOL;
-        //Data
-        csv += contribFragments + Const.EOL;
+                
+        return csvPointsExplanationHeader() + contribFragments + Const.EOL;
+    }
 
-        return csv;
+    private String csvPointsExplanationHeader() {
+        return "In the points given below, an equal share is equal to 100 points. "
+               + "e.g. 80 means \"Equal share - 20%\" and 110 means \"Equal share + 10%\"." + Const.EOL
+               + "Claimed Contribution (CC) = the contribution claimed by the student." + Const.EOL
+               + "Perceived Contribution (PC) = the average value of student's contribution as perceived by the team members."
+               + Const.EOL
+               + "Team, Name, Email, CC, PC, Ratings Recieved" + Const.EOL;
```

Comment:
Alternatively, in the previous method, just return this `+ contribFragments + Const.EOL`.